src/adapter/src/coord.rs

```diff
@@ -500,7 +500,6 @@ pub struct PeekStageExplain {
     select_id: GlobalId,
     finishing: RowSetFinishing,
     df_meta: DataflowMetainfo,
-    used_indexes: UsedIndexes,
```

Comment:
No need to pass `used_indexes: UsedIndexes` explicitly, as this is now recorded directly in the `OptimzierTrace` which is part of the `ExplainContext`.